extra-files/system-tests/dom0.sls

```diff
@@ -18,6 +18,7 @@ dom0-packages:
       - btrfs-progs
       - python3-nose2
       - python3-objgraph
+      - python3-deepmerge
```

Comment:
Wrap this in `{% if grains['osrelease'] >= '4.3' %}` (or just `!= '4.2'`) - the package is not available in F37 (R4.2 dom0).